internal/cmd/testdata/TestKeysListCmd/list_all

```diff
@@ -1,4 +1,4 @@
-  NAME        ISSUED FOR  CREATED       EXPIRES           EXTENSION DEADLINE  
-  front-door  admin       24 hours ago  never             never               
-  side-door   admin       24 hours ago  6 hours from now  26 hours from now   
-  storage     clerk       20 hours ago  6 hours from now  never               
+  NAME        ISSUED FOR  CREATED       LAST USED  EXPIRES           EXTENSION DEADLINE  
+  front-door  admin       24 hours ago  never      never             never               
```

Comment:
Maybe as a follow up, once we've converted the test to use a real API server, but I guess we should add some data that has a last used value, so that we see it formatted in this expected output.

Reply:
I'll make an issue for this. I also want to potentially change the `ISSUED FOR` column to not appear unless you're using the `--all` flag, or the `--user=` argument.